EMML/1001-2000/EMML1939.xml

```diff
@@ -35,51 +35,268 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="p1_i1.8">
+<locus from="102r" to="113v"/>
+<title xml:lang="en" type="complete" ref="LIT6058Ladder">The fifth homily of <persName ref="PRS5721JohnCli">John Climacus</persName> about people who are doing penance</title>
+<incipit xml:lang="gez">በስመ፡ <gap reason="omitted"/>ድርሳን፡ ኀምስ፡ ዘአቡነ፡ ቅዱስ፡ ዮሐንስ፡ በዐለ፡ መዓርግ፡ በእንተ፡ ሰብእ፡ እለ፡ ውስተ፡ ንስሓ፡ ጸሎቶሙ፡ ተሃሉ፡ ምስሌነ፡ አሜን።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="p1_i1.9">
+<locus from="114r" to="123v"/>
+<title xml:lang="en" type="complete" ref="LIT6186JamSarAaron">Homily of <persName ref="PRS5655Jacobof">James of Sarug</persName> on the death of Aaron</title>
+<incipit xml:lang="gez">ድርሳን፡ ዘደረሰ፡ ብፁዕ፡ ወቅዱስ፡ አባ፡ ያዕቆብ፡ ዘስሩግ፡ በእንተ፡ ዕረፍቱ፡ ለአሮን፡ ካህን፡ ወእፎ፡ ነሥኦ፡ ሙሴ፡ እኁሁ፡ ልብሰ፡ ክህነት፡ በሰላመ፡ እግዚአብሔር፡ ጸሎቱ፡
+ወበረከቱ፡ ተሀሉ፡ ምስሌነ፡ አሜን፨</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="p1_i1.10">
+<locus from="124r" to="168r"/>
+<title xml:lang="en" type="complete" ref="LIT2458Testam">History of Joseph</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="p1_i1.10.1">
+<locus from="124r" to="162r"/>
+<title xml:lang="en" type="complete">History of Joseph</title>
+<incipit xml:lang="gez">በስመ፡ <gap reason="omitted"/>ዜናሁ፡ ለዮሴፍ፡ ወልደ፡ ያዕቆብ፡ ወልደ፡ ይስሐቅ፡ ወልደ፡ አብርሃም፡ ወበከመ፡ ሤጥዎ፡ ሎቱ፡ አኃዊሁ፨</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="p1_i1.10.2">
+<locus from="162r" to="168r"/>
+<title xml:lang="en" type="complete">The death of Joseph</title>
+<incipit xml:lang="gez">በስመ፡ እግዚአብሔር፡ መሓሪ፡ ወመስተሣህል፨ ኮነ፡ በውእቱ፡ ዘመን፡ ሶበ፡ በጽሐ፡ መዋዕሊሁ፡ ለዮሴፍ፡ ከመ፡ ይሖር፡ እምዝንቱ፡ ዓለም፡ ኀበ፡ አበዊሁ፡
+ከመ፡ ኵሉ፡ ሰብእ፨</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+</msContents>
+
+<physDesc>
+<handDesc>
+<handNote script="Ethiopic" xml:id="h1"></handNote>
+</handDesc>
+</physDesc>
+
+</msPart>
+
+<msPart xml:id="p2">
+<msIdentifier></msIdentifier>
+<msContents>
+<msItem xml:id="ms_i2">
+<locus from="169r" to="174v"/>
+<title xml:lang="en" type="complete" ref="LIT1700Judges">Judges 1,21-5,26</title>
+</msItem>
+</msContents>
+
+
                <physDesc>
-                  <objectDesc form="Codex">
-                     <supportDesc>
-                        <support>
-                           <material key="parchment"/>
-                        </support>
-                        <extent>
-                           <measure unit="leaf">174 </measure>
-                           <dimensions>
-                              <dim>28 x 20 cm.</dim>
-                           </dimensions>
-                        </extent>
-                        <foliation/>
-                        <condition key="good"/>
-                     </supportDesc>
-                  </objectDesc>
+
+
                   <decoDesc>
                      <decoNote xml:id="d1">
                         <desc>Decorative designs, ff. 2a, 24a, 49a, 60a, 67b, 84a, 86b, 102a, 114a, 124a.</desc>
```

Comment:
Please, check this page for decoration encoding: https://betamasaheft.eu/Guidelines/?q=decoration&id=decorationDescription